data/172/939/589/3/1729395893.geojson

```diff
@@ -20,7 +20,7 @@
     "lbl:longitude":-96.945154,
     "mps:latitude":43.93287,
     "mps:longitude":-96.945154,
-    "mz:hierarchy_label":1,
+    "mz:hierarchy_label":0,
```

Comment:
Same question - how are we determining hierarchy label vs statistical gore vs not having these properties altogether?

Reply:
See above